docs/api_docs/api_doc.md

```diff
@@ -0,0 +1,535 @@
+______________________________________________________________________
```

Comment:
Just to confirm, this is the doc that will be merged into the nebari-docs repo right?

Reply:
And what are the differences between the [PR opened](https://github.com/nebari-dev/nebari/pull/1635/files) on this repo by the Nebari-Sensei bot and this[ PR opened](https://github.com/nebari-dev/nebari-docs/pull/190/files) on the docs repo?

Reply:
yes, this doc is what will go into the actual docs. 

The PR on the docs repo is outdated at this point. After discussion with the team, I need to add an additional GH action to trigger the creating of a PR over on the docs repo.